src/Middleware/RequestQueue/src/Microsoft.AspNetCore.RequestQueue.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>ASP.NET Core middleware for caching HTTP responses on the server.</Description>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <IsAspNetCoreApp>true</IsAspNetCoreApp>
```

Comment:
```suggestion
```
Remove this setting for now. We're not ready to add this assembly to the shared framework.